en/appendices/4-5-migration-guide.rst

```diff
@@ -32,6 +32,35 @@ features::
 A new configuration option has been added to disable deprecations on a path by
 path basis. See :ref:`deprecation-warnings` for more information.
 
+ORM Query API deprecations
+--------------------------
+
+There are some potentially impactful changes to the ORM coming in 5.x. The 4.5
+release introduces deprecations and new ``ORM\Query`` subclasses. The new
+classes offer an API that is consistent with ``ORM\Query`` but emit deprecations
+from all methods that will **not** be present in 5.x.
+
+The new classes are:
+
+- ``Cake\ORM\Query\DeleteQuery`` Used for building ``delete`` queries.
+- ``Cake\ORM\Query\InsertQuery`` Used for building ``insert`` queries.
+- ``Cake\ORM\Query\SelectQuery`` Used for building ``select`` queries.
+- ``Cake\ORM\Query\UpdateQuery`` Used for building ``update`` queries.
+
+Each of these classes no longer offer methods that don't make sense for that
+query type. For example, ``DeleteQuery`` has no ``select()`` clause. In 4.5,
+calling ``select()`` on a query intended to be used as a delete.
```

Comment:
This is missing part of the sentence.

Reply:
There are a few nonsensical things here. I will try to do better in the future.